ebu_tt_live/bindings/__init__.py

```diff
@@ -835,12 +1092,16 @@ def __check_bds(cls, bds):
             )
 
     def toDOM(self, bds=None, parent=None, element_name=None):
+        bds = self.__check_bds(bds)
         xml_dom = super(d_tt_type, self).toDOM(
-            bds=self.__check_bds(bds),
+            bds=bds,
             parent=parent,
             element_name=element_name
         )
-        xml_dom.documentElement.tagName = 'tt:tt'
+        if bds.defaultNamespace() != Namespace:
+            xml_dom.documentElement.tagName = 'tt:tt'
+        else:
+            xml_dom.documentElement.tagName = 'tt'
```

Comment:
Oh this is _that_! When I added that comment I was half expecting to see copy and paste versions of the same thing for every element but since it's just in one place it's not a big deal. Maybe add a comment above explaining what that code is doing and why it is needed, since it looks so hacky?